packages/rolldown/tests/fixtures/plugin/transform/_config.ts

```diff
@@ -8,10 +8,11 @@ export default defineTest({
     plugins: [
       {
         name: 'test-plugin',
-        transform: function (code, id) {
+        transform: function (code, id, module_type) {
           transformFn()
           if (id.endsWith('foo.js')) {
             expect(code).toStrictEqual('')
+            expect(module_type).toEqual('js')
```

Comment:
```suggestion
            expect(meta.module_type).toEqual('js')
```

Looks more suitable.